05_pytorch_backend/cnn_acceleration_tensorly_and_pytorch.ipynb

```diff
@@ -0,0 +1,594 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accelerating deep neural networks with tensor decompositions\n",
+    "\n",
+    "In this notebook we will show how to use TensorLy for accelerating convolutional layers in PyTorch.\n",
+    "\n",
+    "This notebook is based on this blog post: https://jacobgil.github.io/deeplearning/tensor-decompositions-deep-learning\n",
+    "\n",
+    "We will use the Tucker and CP decomositions to make low rank approximations of convolutional layers, and make them:\n",
+    "1. Faster.\n",
+    "2. Smaller.\n",
+    "\n",
+    "Like pruning, these are tools that can be used to make the networks smaller and faster, and then try to restore accuracy by training further.\n",
```

Comment:
Typo: (decomositions instead of decomposition) 
Also the two sentences are slightly redundant, we could replace by a similar paragraph such as:

We will show how to use Tucker and CP decompositions to find low rank approximations of convolutional layers.
Akin to prunning, this will allow use to reduce the number of parameters as well as make them faster. While the low-rank approximation can decrease the accuracy, this can be restored by training further.